content/en/docs/setup/install/helm/index.md

```diff
@@ -157,6 +161,20 @@ $ helm template install/kubernetes/helm/istio --name istio --namespace istio-sys
 
 {{< /tab >}}
 
+{{< tab name="DNS certificates" cookie-value="dnscerts" >}}
+
+Install Istio with two example DNS certificates signed by Kubernetes CA:
+
+{{< text bash >}}
+$ helm template \
+    --name=istio \
+    --namespace=istio-system \
+    --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
+    install/kubernetes/helm/istio | kubectl apply -f -
+{{< /text >}}
+
+{{< /tab >}}
+
 {{< /tabset >}}
 
 ### Option 2: Install with Helm and Tiller via `helm install`
```

Comment:
We've deprecated helm installation instructions for 1.4, so this section can go I think.

@frankbu ?

Reply:
Done: changed to use istioctl to install Istio.